build/tests.webpack.js

```diff
@@ -1,3 +1,3 @@
-var context = require.context('../web', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
+var context = require.context('../web/client/components/geostory', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
```

Comment:
do not commit tests.webpack.js
```suggestion
var context = require.context('../web', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
```